bindings/wasm/CMakeLists.txt

```diff
@@ -17,8 +17,12 @@ project(wasm)
 add_executable(manifoldjs bindings.cpp)
 
 # make sure that we recompile the wasm when bindings.js is being modified
-set_source_files_properties(bindings.cpp OBJECT_DEPENDS
-  ${CMAKE_CURRENT_SOURCE_DIR}/bindings.js)
+set_source_files_properties(bindings.cpp
+  OBJECT_DEPENDS ${CMAKE_CURRENT_SOURCE_DIR}/bindings.js
+  # FIXME: can we rewrite the custom command to run after these are modified when building
+  # without recompiling bindings.cpp?
```

Comment:
Probably, but my CMake foo is weak. Feel free to play around!

Reply:
I believe that I've got something working for the interface copying that does not require rebuilding the bindings needlessly. :+1: